frontend/src/app/shop/order-menu/order-menu.component.html

```diff
@@ -106,14 +112,27 @@
               </div>
             </div>
 
-            <div class="flex gap-1 items-center">
+            <!-- <div class="flex gap-1 items-center">
               <img
                 [src]="product.image"
                 [alt]="product.name"
                 class="w-[65px] h-[65px]" />
               <div>
                 {{ product.description }}
               </div>
+            </div> -->
+
+            <!-- Easter Egg product display logic -->
+            <div class="flex gap-1 items-center">
+              <ng-container *ngIf="product.description.toLowerCase().includes('easter egg')">
```

Comment:
if description is not filled: ..description?.toLowerC...

Reply:
Great catch!